specification.md

```diff
@@ -1830,6 +1842,7 @@ message OpAMPConnectionSettings {
     string destination_endpoint = 1;
     Headers headers = 2;
     TLSCertificate certificate = 3;
+    optional uint64 heartbeat_interval_seconds = 4;
```

Comment:
This is potentially an overkill/nit, but do you think we should use a [duration](https://protobuf.dev/reference/protobuf/google.protobuf/#duration) as the type?

We already have another field that is a duration but uses different units (`retry_after_nanoseconds`) which is potentially confusing. This is also in-line with [Google's AIPs](https://google.aip.dev/142).